addons/payment_payulatam/models/__init__.py

```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from . import payment
+from . import payment_acquirer
+from . import payment_transaction
```

Comment:
It seems that those two files were lost in the git process :D

Reply:
:man_facepalming: